src/main/java/com/checkmarx/jenkins/CxScanBuilder.java

```diff
@@ -1296,7 +1296,11 @@ private CxScanConfig resolveConfiguration(Run<?, ?> run, DescriptorImpl descript
         log.info("  ORIGIN FROM JENKIN :: " + jenkinURL);
         log.info("  ORIGIN URL FROM JENKIN :: " + originUrl);
 
-        ret.setPostScanActionId(getPostScanActionId());
+        if(getPostScanActionId() == 0)
+        	ret.setPostScanActionId(null);
+        else
+        	ret.setPostScanActionId(getPostScanActionId());
```

Comment:
Please, make it optional @umeshwaghode

Reply:
@umeshwaghode  After we upgraded Checkmarx plugin to Release_2022.1.3, below exception occured if we do not set the postScanActionId.
`"java.lang.Integer.intValue()" because the return value of "com.checkmarx.jenkins.CxScanBuilder.getPostScanActionId()" is null`

As a workaround, we set postScanActionId to 0, but this is not compatible with the OPTIONAL definition of postScanActionId.